p192/src/ecdsa.rs

```diff
@@ -13,29 +13,6 @@
 //! - `ecdsa`: provides `ecdsa-core` features plus the [`SigningKey`] and
 //!   [`VerifyingKey`] types which natively implement ECDSA/P-192 signing and
 //!   verification.
-//!
```

Comment:
What to do about this, impossible to obtain the signing key without signing support, should we remove this or use a static set example key?

Reply:
Remove it from the docs